apps/librelingo_json_export/tests/test_cli.py

```diff
@@ -86,3 +96,18 @@ def test_dry_run_doesnt_create_output_files(
 def test_dry_run_calls_real_export_course(mocks, inputs, invoke):
     invoke([*inputs, "--dry-run"])
     assert mocks["export_course"].call_count == 1
+
+
+@pytest.mark.parametrize("no_dry_run_arg", [["--no-dry-run"], []])
+def test_creates_excpected_files(fs, invoke_with_no_mocks, no_dry_run_arg):
+    input_path = Path(__file__).parent / "fixtures" / "fake_course"
+    fs.add_real_directory(input_path)
+    output_path = fakes.path()
+    assert not output_path.exists()
```

Comment:
I think this assertion is not important or redundant, actually we don't really care if that folder exists, right? besides it should logically follow that it doesn't exist.

Reply:
Since `fs` is explicitly as one of the arguments, I agree.